packages/api-nest/src/reading-progress/reading-progress.service.ts

```
@@ -0,0 +1,157 @@
+
+  /**
+   * Get completion percentage based on sentinel progress
+   * @param progress - Reading progress entity
+   * @param totalSentinels - Total number of sentinels in the content
+   * @returns Completion percentage (0-100)
+   */
+  calculateCompletion(
+    progress: ReadingProgressEntity,
+    totalSentinels: number,
+  ): number {
+    if (totalSentinels <= 0) return 0
```

ESLint

[error] Expected { after 'if' condition.

(curly)